commands/info.md

```diff
@@ -83,10 +83,10 @@ Here is the meaning of all fields in the **clients** section:
 *   `blocked_clients`: Number of clients pending on a blocking call (`BLPOP`,
      `BRPOP`, `BRPOPLPUSH`, `BLMOVE`, `BZPOPMIN`, `BZPOPMAX`)
 *   `tracking_clients`: Number of clients being tracked (`CLIENT TRACKING`)
-*   `pubsub_clients`: Number of clients in pubsub mode (`SUBSCRIBE`, `PSUBSCRIBE`, `SSUBSCRIBE`). Added in Valkey 8.0
-*   `watching_clients`: Number of clients in watching mode (`WATCH`). Added in Valkey 8.0
+*   `pubsub_clients`: Number of clients in pubsub mode (`SUBSCRIBE`, `PSUBSCRIBE`, `SSUBSCRIBE`).
+*   `watching_clients`: Number of clients in watching mode (`WATCH`).
 *   `clients_in_timeout_table`: Number of clients in the clients timeout table
-*   `total_watched_keys`: Number of watched keys. Added in Valkey 8.0.
+*   `total_watched_keys`: Number of watched keys.
```

Comment:
I'm strongly against this change.

Where is the decision to remove all history? Please show me. 

It is a terrible idea for anyone who is upgrading from Redis, which I think will be 99% of our users.

We insist that we keep references to Redis 6.2 and later, i.e. the still supported versions when we forked.

@valkey-io/core-team I would like to know where this idea is coming from. Was it discussed anywhere?

Reply:
yes, i think we should keep the history, finding the corresponding supported version is a pain (it deserves to be properly documented at somewhere). I don't remember us discussing this.

Reply:
We didn't discuss this in the core team meeting but there was a discussion on an early re-branding PR (and this predates @enjoy-binbin's time on this project).

https://github.com/valkey-io/valkey-doc/pull/4#discussion_r1548636459


IIRC, the decision back then is to keep anything beyond 7.2 (the origin of Valkey) and this is why 8.0 was kept. 

https://github.com/valkey-io/valkey-doc/pull/4#discussion_r1548638458

>I insist that we keep references to Redis 6.2 and later

I am not against keeping references to Redis 6.2+ but can you explain how it could help customers who switch over to Valkey?

Reply:
Some users avoid upgrading until it is absolutely necessary. When a version is end of life, users want to switch to the latest version. That means from redis 6.2 to valkey 7.2 or valkey 8.

Reply:
I can't see in that PR that there is a decision to keep anything beyond 7.2.

* Some really old stuff is deleted, like redis 2 (that's good)
* Valkey 8 is kept. This is good and important, since it is a future version and not yet released. It's important to keep this to indicate that these parts are not valid for our first release 7.2.5.
* Delete references to 7.2, 7.0, 6.2 is unfortunate but no disaster. I think we should avoid it. We don't have versioned docs so these indicators is the only way for users to find these differences when upgrading, apart from the release notes in each release.

Reply:
>Some users avoid upgrading until it is absolutely necessary. When a version is end of life, users want to switch to the latest version. That means from redis 6.2 to valkey 7.2 or valkey 8.

Got it - I think we should re-instate the history of these versions  then. I am all for helping the users.